shuup/front/basket/command_dispatcher.py

```diff
@@ -73,8 +74,9 @@ def handle(self, command, kwargs=None):
         except (Problem, ValidationError) as exc:
             if not self.ajax:
                 raise
+            msg = exc.message if hasattr(exc, "message") else exc
             response = {
-                "error": force_text(exc, errors="ignore"),
+                "error": force_text(msg, errors="ignore"),
```

Comment:
Checked that this will make a difference with ValidationErrors, if the validation error is simple one-field only error (i.e. not a list of errors or dict of errors). So I guess this is better now. No need for changes to the PR.